sdk/core/azure-core/src/main/resources/META-INF/native-image/com.azure/azure-core/reflect-config.json

```diff
@@ -136,5 +136,26 @@
     "allPublicConstructors": true,
     "allDeclaredMethods": true,
     "allPublicMethods": true
+  },
+  {
+    "name":"com.azure.core.models.ResponseError",
+    "allDeclaredConstructors": true,
+    "allPublicConstructors": true,
+    "allDeclaredMethods": true,
+    "allPublicMethods": true
+  },
+  {
+    "name":"com.azure.core.models.ResponseInnerError",
+    "allDeclaredConstructors": true,
+    "allPublicConstructors": true,
+    "allDeclaredMethods": true,
+    "allPublicMethods": true
+  },
```

Comment:
I am not sure why `ResponseError` and `ResponseInnerError` was not in reflect-config.json